crates/bevyhavior_simulator/src/robot.rs

```diff
@@ -333,7 +306,7 @@ pub fn move_robots(mut robots: Query<&mut Robot>, mut ball: ResMut<BallResource>
 
 #[derive(Event, Clone, Copy)]
 pub struct Message {
-    pub sender: PlayerNumber,
+    pub sender: u8,
```

Comment:
The whole point of this PR is to get away from the ever expanding enum :man_facepalming: :fearful: :sob: so that we could play with any give jersey numbers without having to constantly worry about matching an enum. But I could do a type alias. Where do you think it makes the most sense to put it?

Reply:
somewhere in types.rs I guess?